src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java

```diff
@@ -38,6 +38,12 @@ public ResponseEntity<Void> addAudioRequest(AudioRequestDetails audioRequestDeta
         return new ResponseEntity<>(HttpStatus.CREATED);
     }
 
+    public ResponseEntity<Void> deleteAudioRequest(Integer audioRequestId) {
```

Comment:
If Hemanta agrees with the base path being `audio-requests` pluralised - then I think the new services we do this sprint should go under `AudioRequestsController`